packages/manager/src/features/Users/UserPermissionsEntitySection.tsx

```diff
@@ -54,61 +58,79 @@ export const UserPermissionsEntitySection = React.memo(
     };
 
     return (
-      <Box key={entity} paddingBottom="0" marginTop={`${theme.spacing(2)}`}>
+      <Box key={entity} marginTop={`${theme.spacing(2)}`} paddingBottom="0">
         {showHeading && (
           <Typography
-            variant="h3"
             sx={{
-              marginTop: theme.spacing(3),
               marginBottom: theme.spacing(2),
+              marginTop: theme.spacing(3),
             }}
             data-qa-permissions-header={entity}
+            variant="h3"
           >
             {grantTypeMap[entity]}
           </Typography>
         )}
         <StyledGrantsTable aria-label="User Permissions" noBorder>
           <TableHead data-qa-table-head>
-            <TableRow>
+            <TableRow
+              sx={(theme) => ({
+                'span.MuiFormControlLabel-label': {
+                  fontFamily: theme.font.bold,
+                },
+              })}
+            >
               <TableCell>Label</TableCell>
               <TableCell padding="checkbox">
-                {/* eslint-disable-next-line */}
-                <label style={{ marginLeft: -35, cursor: 'pointer' }}>
-                  None
-                  <Radio
-                    name={`${entity}-select-all`}
-                    checked={entityIsAll(null)}
-                    value="null"
-                    onChange={entitySetAllTo(entity, null)}
-                    data-qa-permission-header="None"
-                  />
-                </label>
+                <FormControlLabel
+                  control={
+                    <Radio
+                      inputProps={{
+                        'aria-label': `${entity}s, set-all-permissions-to-none`,
+                      }}
+                      checked={entityIsAll(null)}
+                      data-qa-permission-header="None"
+                      name={`${entity}-select-all`}
+                      onChange={entitySetAllTo(entity, null)}
+                      value="null"
+                    />
+                  }
+                  label="None"
```

Comment:
Rather than using the `span.MuiFormControlLabel-label` selector on the TableRow, we could apply the bold style here.

Honestly not sure which is better. Sometimes I'm hesitant with using MUI classnames because they could change in future MUI releases and cause the UI to break


```suggestion
                  slotProps={{
                    typography: {
                      sx: (theme) => ({ fontFamily: theme.font.bold }),
                    },
                  }}
                  label="None"
```